docs/publishing/index.md

```diff
@@ -0,0 +1,123 @@
+
+Keep in mind that these suggestions are not specifically tuned for publishing to a CDN or other contexts where broader application-specific optimizations will not be applied to the code available therein.
+
+<style>
+.content__default ul {list-style:none;padding:0;}
+</style>
+
+### Do: 👍
+
+- ✅ [Do publish latest standard EcmaScript](#do-publish-latest-standard-ecmascript)
+- ✅ [Do publish standard es modules](#do-publish-standard-es-modules)
+- ✅ [Do include appropriate `package.json` entries](#do-include-appropriate-packagejson-entries)
+- ✅ [Do export element classes](#do-export-element-classes)
+- ✅ [Do export side effects separately](#do-export-side-effects-separately)
+- ✅ [Do keep side effects out of your index.js](#do-keep-side-effects-out-of-your-indexjs)
+- ✅ [Do import 3rd party node modules with "bare" import specifiers](#do-import-3rd-party-node-modules-with-bare-import-specifiers)
+- ✅ [Do include file extensions in import specifiers, always](#do-include-file-extensions-in-import-specifiers-always)
+- _Optional_: [use `type: "module"` when possible](#optional-use-type-module)
+- _Optional_: [include typings; e.g. `*.d.ts` files](#optional-include-typings)
+- _Optional_: [include source maps](#optional-include-source-maps)
+
+### Don't: 👎
+
+- ❌ [Do not bundle](#do-not-bundle)
+- ❌ [Do not minify](#do-not-minify)
+- ❌ [Do not optimize](#do-not-optimize)
+- ❌ [Do not use `.mjs` file extensions](#do-not-use-mjs-file-extensions)
+- ❌ [Do not import polyfills](#do-not-import-polyfills)
+
+## Do publish latest standard EcmaScript
+
+Contrary to some long-standing beliefs, an implementing project will always know more about the delivery context of the code it leverages than the developer of any of its dependencies. Taking this fact into account, the act of transpiling your code below a reasonably modern level (e.g. [TC39 Stage 4](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and/or available cross-browser) is the best way that you can prepare your package to empower downstream projects to make their own decisions about the code they ship to their users.
+
+In some cases, you may choose to use newer specifications that may require special care when preparing for older browsers. In this case, transpiling down to the level you choose from above may be a good idea. If you choose not to do this, be sure to document this reality thoroughly so that your users are not left unawares.
+
+## Do publish standard es modules
+
+Your package (and those packages that it depends on) should be distributed as standard es modules. This supports the "do" above to deliver the latest standard EcmaScript to your users, modules are both Stage 4 and supported in all major browsers, but also prepares your users to have the simplest time building their applications for production as es modules are the only module format natively supported by browsers.
```

Comment:
```suggestion
Your package (and the packages that it depends on) should be distributed as standard es modules. This supports the "do" above to deliver the latest standard EcmaScript to your users, modules are both Stage 4 and supported in all major browsers, but also prepares your users to have the simplest time building their applications for production as es modules are the only module format natively supported by browsers.
```

> as es modules are the only module format natively supported by browsers.

as opposed to?

Reply:
CJS, AMD, UMD? Stole that a little from Justin's article. Will rethink.

Reply:
Maybe its fine to just leave it out